### DISCUSSION TITLE
Saving text data and audio data for a trial ~ No plugin loaded for trials of type "call-function"

### DISCUSSION THREAD

**Discussion by Swatermx** [jspsych/jsPsych#2383] [Q&A] [answered]
Hiya!!

I was wondering if anyone was able to help me with an error I get when I run my experiment. It ran when I didn't have data saving externally to an excel .csv file, but now I'm getting the error 
"No plugin loaded for trials of type "call-function" " and 
"Uncaught TypeError: jsPsych.plugins[trial.type] is undefined ". I was wondering what these meant, or if anyone knew how to fix it? 
I can add code if that's helpful, just wouldn't know what would be the useful part.

Any help would be much appreciated!!

**Comment by jodeleeuw** [comment, marked as answer]
Hi @Swatermx,

The "No plugin loaded ..." error almost always means that the `.js` file for the plugin isn't included in the experiment. You may need to add a `<script>` tag to load it. 

Here's [a relevant section from the tutorial](https://www.jspsych.org/7.1/tutorials/hello-world/#step-4-use-a-plugin-to-print-a-message).
